src/Avalonia.Base/Threading/DispatcherPriorityAwaitable.cs

```diff
@@ -1,40 +1,59 @@
 using System;
 using System.Runtime.CompilerServices;
-using System.Threading.Tasks;
 
 namespace Avalonia.Threading;
 
-public class DispatcherPriorityAwaitable : INotifyCompletion
```

Comment:
Breaking change

Reply:
By the way, Max mentioned that it wasn't caught by the pipeline when it should.

Reply:
That's because API validator compares current assembly to 11.0.0.
This is wrong and should be fixed, but not now.

Reply:
Current builds target 11.1, and this should be marked as a proper breaking change now.